diagnosis/getkuberneteslogs.sh

```diff
@@ -230,7 +202,9 @@ then
         echo "[$(date +%Y%m%d%H%M%S)][INFO] Processing host $host"
         
         echo "[$(date +%Y%m%d%H%M%S)][INFO] Uploading scripts"
-        scp -q -r $SCRIPTSFOLDER/*.sh $USER@$host:/home/$USER/
+        scp -q -r common.sh $USER@$host:/home/$USER/
```

Comment:
Need to remove reference to the SCRIPTSFOLDER from line 185

Reply:
yeah, just saw it, bad rebase